File: src/main/java/com/matamercer/microblog/Exceptions/NotFoundException.java
package com.matamercer.microblog.Exceptions;

import lombok.Getter;
import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(value = HttpStatus.NOT_FOUND)
@Getter
public class NotFoundException extends RuntimeException{
    private String message;

    public NotFoundException(){

    }

    public NotFoundException(String message){
        this.message = message;
    }
}

File: src/main/java/com/matamercer/microblog/exceptions/NotFoundException.kt
package com.matamercer.microblog.exceptions

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(value = HttpStatus.NOT_FOUND)
class NotFoundException : RuntimeException {
    override var message: String? = null

    constructor()
    constructor(message: String?) {
        this.message = message
    }
}